src/main/java/uk/ac/ebi/eva/submission/util/EmailNotificationHelper.java

```diff
@@ -44,6 +44,23 @@ public String getTextForSubmissionStatusUpdate(SubmissionAccount submissionAccou
         return notificationText;
     }
 
+    public String getTextForEVAHelpdeskSubmissionUploaded(String submissionId, String projectTitle) {
+        String notificationText = new HTMLHelper()
+                .addText("Dear EVA Helpdesk,")
+                .addGap(1)
+                .addText("The user has uploaded a new Submission: ")
```

Comment:
We should add the user's first and last name and the primary email address so the user can be contacted if there are any question.

Reply:
added